### DISCUSSION TITLE
Изменить стиль для редактируемых полей на IndexPage (updateOnPreview)

### DISCUSSION THREAD

**Discussion by ramir1** [moonshine-software/moonshine#1181] [Ideas]
Убрать иконку, а вместо этого сделать подчеркивание пунктиром: text-decoretion: dashed
Это сделает таблицу компактнее. Особенно, если полей много.

**Comment by lee-to** [comment]
Сделали так в 3 версии и само редактирование в попапе но в 2 уже не будем переносить
